### PR TITLE
Get Ready post

### PR DESCRIPTION
**Pull request by ericholscher**
Also updates social event venue,
and fixes a couple other unconf room issues

----
:books: Documentation preview :books:: https://writethedocs-www--1938.org.readthedocs.build/

### CODE REVIEW

File: docs/conf/portland/2023/news/events-activities.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+.. post:: April 28, 2023
+   :tags: tickets, events, practical, shirts, {{shortcode}}-{{year}}
+
+Events and Activities, Hike Update, and Livestream
+==================================================
+
+We're less than 2 weeks out from the conference, and we hope everyone is getting excited!
+
+In this update, we're happy to share with you the different events and activities that will be available as a part of the conference,
+discuss a change to the hike, and share details on the talk livestream.
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/events-activities.rst]:
```suggestion
discuss a change to the hike, and share details on the livestream.
```

I think that actually makes it less clear

File: docs/conf/portland/2023/news/events-activities.rst [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Remember to check out our `Welcome Wagon <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/welcome-wagon/>`_ guide and the `Visiting {{city}} <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/visiting/>`_ page for information around transit, food, and making the most out of the conference.
+
+Hike Reroute
+------------
+
+Unfortunately, there is a closure of the trail that we were planning to hike because of a large rain event this spring.
+This means the hike will turn into an "out and back" hike from Lower Macleay Trailhead to Pittock Mansion.
+The options for public transit at the bottom of the trail are a bit limited,
+so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/events-activities.rst]:
```suggestion
Unfortunately the trail that we were planning to hike is currently closed due to spring rain.
This means that we'll hike "out and back" from Lower Macleay Trailhead to Pittock Mansion, and returning to the starting point.
Options for public transit to and from the starting point are a bit limited,
so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
```

File: docs/conf/portland/2023/news/events-activities.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+See more information on the `Hike page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/outing/>`_
+
+Live streaming
+--------------
+
+If you can't attend the conference in person, we will be broadcasting all the main stage presentations on our `livestream page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/livestream>`_ .
+You can join the #live-streaming channel on Slack to connect with other remote attendees, ask questions, and troubleshoot technical issues with the stream.
+We are excited to be able enable folks to participate in the conference from afar!
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/events-activities.rst]:
```suggestion
We are excited that folks can watch the conference talks from anywhere in the world with an internet connection!
```

**Review by plaindocs — COMMENTED** [review]
Reviewed as far as Events and Activities.

File: docs/conf/portland/2023/outing.rst [hunk 4]
```diff
@@ -28,22 +28,57 @@ Logistics
...
-**Note**
+Hike Reroute
+------------
 
-The hike does not end where it starts. Take this into consideration when you plan.
-There is an out-and-back option if you choose to just go to Pittock Mansion, then return back down to the start.
+Unfortunately, there is a closure of the trail that we were planning to hike because of a large rain event this spring.
+This means the hike will turn into an "out and back" hike from Lower Macleay Trailhead to Pittock Mansion.
+The options for public transit at the bottom of the trail are a bit limited,
+so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
```
Comment (plaindocs) [inline docs/conf/portland/2023/outing.rst]:
```suggestion
Unfortunately the trail that we were planning to hike is currently closed due to spring rain.
This means that we'll hike "out and back" from Lower Macleay Trailhead to Pittock Mansion, and returning to the starting point.
Options for public transit to and from the starting point are a bit limited, so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
```

File: docs/conf/portland/2023/outing.rst [hunk 5]
```diff
@@ -28,22 +28,57 @@ Logistics
...
 
-{% include "conf/portland/hike.rst" %}
+What to bring
+~~~~~~~~~~~~~
 
-{% else %}
+May in Portland can be interesting. It will probably be in the 60s, with a chance of rain. So, please bring:
```
Comment (plaindocs) [inline docs/conf/portland/2023/outing.rst]:
```suggestion
May in Portland can be interesting. The temperature will probably be in the 60s, with a chance of rain. So, please bring:
```

File: docs/conf/portland/2023/venue.rst [hunk 6]
```diff
@@ -80,12 +80,13 @@ Parking
 *******
 
 There are two ADA spots in the guest parking lot and there is no charge to use these spots.
+Any of the streets east of the venue (15th Ave and higher) allow for all-day street parking.
 
 Entering the building
 *********************
 
 The main entrance is actually an assuming ground-level door on the west side of building.
```
Comment (plaindocs) [inline docs/conf/portland/2023/venue.rst]: Assuming? Or unassuming?

**Review by plaindocs — APPROVED** [review]
Looks good. Few nits to fix before shipping.

File: docs/conf/portland/2023/venue.rst (hunk 6 above)
Comment (ericholscher) [reply docs/conf/portland/2023/venue.rst]: Unassuming heh

```suggestion
The main entrance is actually an unassuming ground-level door on the west side of building.
```

File: docs/conf/portland/2023/news/welcome-get-ready.rst [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+In this update we discuss a change to the hike, share details on the livestream, and a few other updates as the conference approaches.
+
+Hike Reroute
+------------
+
+Unfortunately, there is a closure of the trail that we were planning to hike because of a large rain event this spring.
+This means the hike will turn into an "out and back" hike from Lower Macleay Trailhead to Pittock Mansion.
+The options for public transit at the bottom of the trail are a bit limited,
+so we will work to ensure folks have a ride, or can get a ride share back to your accommodation.
+
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/welcome-get-ready.rst:21]: You reverted my suggested change to this, intnentionally?

File: docs/conf/portland/2023/news/welcome-get-ready.rst [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+Live streaming
+--------------
+
+If you can't attend the conference in person, we will be broadcasting all the main stage presentations on our `Livestream page <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/livestream/>`_ .
+You can join the #live-streaming channel on Slack to connect with other remote attendees, ask questions, and troubleshoot technical issues with the stream.
+We are excited to be able enable folks to participate in the conference from afar!
+
+Be prepared
+-----------
```
Comment (plaindocs) [inline docs/conf/portland/2023/news/welcome-get-ready.rst]:
```suggestion
Things to prepare for
----------------------------
```

sounds less ominous to me

File: docs/include/conf/coc/na-coc.rst [hunk 9]
```diff
@@ -1,14 +1,10 @@
...
-**Jennifer Rondeau**
-
-* jennifer.rondeau@gmail.com
-* slack: jrondeau
-
 **Sasha Romijn**
 
 * sasha@writethedocs.org
-* slack: sasha
+* slack: sasha
```
Comment (plaindocs) [inline docs/include/conf/coc/na-coc.rst]: Might want this to be me not @mxsasha cos timezones?

**Review by plaindocs — APPROVED** [review]
LGTM

File: docs/conf/portland/2023/news/welcome-get-ready.rst (hunk 7 above)
Comment (ericholscher) [reply docs/conf/portland/2023/news/welcome-get-ready.rst:21]: Bleh, no. No sure how they got lost :(

Comment (ericholscher) [reply docs/conf/portland/2023/news/welcome-get-ready.rst:21]: Re-added.

**Pull request merged**